extern/CMakeProject-libusb.cmake

```diff
@@ -0,0 +1,20 @@
+# Build and statically link libusb project
+
+# libusb is currently used by Linux, but we need to test it under windows too
+# and it should work
+if(NOT LINUX)
+  return()
+endif()
+
+include(ExternalProject)
+ExternalProject_Add(
+  libusb
+  GIT_REPOSITORY "https://github.com/libusb/libusb-cmake.git"
```

Comment:
One reason I do prefer having the library already in extern is mostly for the benefit of being able to compile the game when offline, which I personally find myself doing pretty often when I'm on the go. Do you know if it would be possible to do that?

Reply:
Ok, I see your point. Once it built, git repo cloned into the `Build` directory. But after clean it require clone again.
That is not you expect. If offline build is so important I can consider and refactor it into a git submodule. Yet personally I don't like submodules.

Reply:
> Yet personally I don't like submodules.

I understand why, but it is important to both myself and teejusb that the game can easily be compiled without an internet connection.